package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "car-rental-front-end",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
```

Comment:
- It looks like there isn't a command to run `npm run test`. I kindly invite you to add it, this could help to run tests and check everything is allright for this.